drivers/scheduler/k8s/specs/postgres-csi/azure/azure-storage-class.yaml

```diff
@@ -0,0 +1,11 @@
+##### Azure storage class
```

Comment:
can you please maintain different folder for azure , like postgres-cephfs-csi

since we were facing issue with the current folder structure you have created

Reply:
Done, Create a spec dir potsgres-azure-disk